src/app/static/src/app/components/plots/MapAdjustScale.vue

```diff
@@ -78,12 +80,14 @@ import {ColourScaleType} from "../../store/colourScales/colourScales";
     interface Props {
         show: boolean,
         colourScale: ColourScaleSettings,
-        step: number
+        step: number,
+            metadata: any
```

Comment:
could you also do an auto-format on this file? spacing has gone a bit weird

Reply:
I'm struggling to configure my autoformatter so that it doesn't but every html attribute on a separate line so I cannot format without making big changes to the file.